libevmasm/Instruction.h

```diff
@@ -281,19 +281,22 @@ inline Instruction logInstruction(unsigned _number)
 	return Instruction(unsigned(Instruction::LOG0) + _number);
 }
 
+/// Gas price tiers representing static cost of an instruction.
+/// Opcodes whose cost is dynamic or depends on EVM version should use the `Special` tier and need
+/// dedicated logic in GasMeter (especially in estimateMax()).
+/// The tiers loosely follow opcode groups originally defined in the Yellow Paper.
 enum class Tier
 {
-	Zero = 0,	// 0, Zero
-	Base,		// 2, Quick
-	VeryLow,	// 3, Fastest
-	Low,		// 5, Fast
-	Mid,		// 8, Mid
-	High,		// 10, Slow
-	Ext,		// 20, Ext
-	ExtCode,	// 700, Extcode
-	Balance,	// 400, Balance
```

Comment:
I'm removing these tiers because they seem obsolete. The instructions using them no longer have a static cost independent of the EVM version so I think they should be using the `Special` tier.